pom.xml

```
@@ -291,7 +291,7 @@
         <dependency>
             <groupId>org.hisp.dhis.parser</groupId>
             <artifactId>dhis-antlr-expression-parser</artifactId>
-            <version>1.0.22</version>
+            <version>1.0.23</version>
```

Antlr parser version is already to 1.0.24

```suggestion
            <version>1.0.24</version>
```